fe/fe-core/src/main/java/com/starrocks/authentication/AuthenticationMgr.java

```diff
@@ -171,11 +172,19 @@ public boolean doesUserExist(UserIdentity userIdentity) {
         }
     }
 
+    public long getMaxConn(UserIdentity currUserIdentity) {
```

Comment:
Can you add some comments in the code for this function for better understand the differences between it and the other getMaxConn.

Reply:
done